### PR TITLE
refactor: move generated_symbol_exports_list_* files to napi folder

### PR DESCRIPTION
**Pull request by dsherret**
These files are adding noise to the main cli folder. Let's move them into the `napi` folder since they're specific to that.

### CODE REVIEW

File: cli/build.rs [hunk 1]
```diff
@@ -351,7 +351,7 @@ fn main() {
     panic!("Cross compiling with snapshot is not supported.");
   }
 
-  let symbols_path = std::path::Path::new(
+  let symbols_path = std::path::Path::new("napi").join(
```
Comment (bartlomieju) [inline cli/build.rs:354]: Last time we did that it broke the release/publish process.

Comment (dsherret) [reply cli/build.rs:354]: I think it was outside the CLI folder before?

Comment (dsherret) [reply cli/build.rs:354]: Oh, it was in napi_sym: https://github.com/denoland/deno/pull/16330/files#diff-f744e98f02b8c530a278d32cdb7dd6fed387c0ae06607944b61324d0af3b7697 -- I think this should be ok.

Comment (bartlomieju) [reply cli/build.rs:354]: Okay, let's try that

**Review by bartlomieju — APPROVED** [review]

**Pull request merged**